### ISSUE TITLE
Hosted Engine Deployment fails on CPU model name

### ISSUE THREAD

**Issue by tim427** [oVirt/ovirt-ansible-collection#284]
##### SUMMARY
Hosted Engine Deployment fails on CPU model name

##### COMPONENT NAME
`01_create_target_hosted_engine_vm.yml` -> `Convert CPU model name`
https://github.com/oVirt/ovirt-ansible-collection/blob/63c2d1496e46b92583116ae7d39c187cb2cb4a62/roles/hosted_engine_setup/tasks/create_target_vm/01_create_target_hosted_engine_vm.yml#L64

##### STEPS TO REPRODUCE

- Fresh install of Centos 8.3 Stream
- Cockpit enabled
- oVirt repo's added
- GlusterFS deployed
- Hosted Engine Deployment wizard
- Add GlusterFS as storage
- Deploy

##### EXPECTED RESULTS
Successful deployment of a Hosted Engine

##### ACTUAL RESULTS

```
[ INFO ] TASK [ovirt.ovirt.hosted_engine_setup : Convert CPU model name]
[ ERROR ] fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute ''\n\nThe error appears to be in '/usr/share/ansible/collections/ansible_collections/ovirt/ovirt/roles/hosted_engine_setup/tasks/create_target_vm/01_create_target_hosted_engine_vm.yml': line 69, column 15, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n - debug: var=server_cpu_dict\n ^ here\n\nThere appears to be both 'k=v' shorthand syntax and YAML in this task. Only one syntax may be used.\n"}
```

**Comment by arachmani** [comment]
Hi @tim427,
Can you please share ovirt-hosted-engine-setup logs and libvirt version?
It's probably related to https://bugzilla.redhat.com/show_bug.cgi?id=1961558.

**Comment by tim427** [comment]
Sure! I'll try a new deployment next week, to also verify https://github.com/oVirt/ovirt-ansible-collection/issues/283

Keep you posted!

**Comment by arachmani** [comment]
Following https://bugzilla.redhat.com/show_bug.cgi?id=1968082#c7, this issue has been fixed in libvirt-7.4.0-1.el8s.x86_64.
